src/fidesops/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -0,0 +1,246 @@
+from __future__ import annotations
+
+import logging
+
+from fastapi import Depends, HTTPException
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.orm import Session
+from starlette.status import (
+    HTTP_200_OK,
+    HTTP_400_BAD_REQUEST,
+    HTTP_403_FORBIDDEN,
+    HTTP_404_NOT_FOUND,
+    HTTP_500_INTERNAL_SERVER_ERROR,
+)
+
+from fidesops.ops.api.deps import get_db
+from fidesops.ops.api.v1.urn_registry import (
+    CONSENT_REQUEST,
+    CONSENT_REQUEST_PREFERENCES,
+    CONSENT_REQUEST_VERIFY,
+    V1_URL_PREFIX,
+)
+from fidesops.ops.common_exceptions import (
+    EmailDispatchException,
+    FunctionalityNotConfigured,
+    IdentityVerificationException,
+)
+from fidesops.ops.core.config import config
+from fidesops.ops.models.privacy_request import (
+    Consent,
+    ConsentRequest,
+    ProvidedIdentity,
+    ProvidedIdentityType,
+)
+from fidesops.ops.schemas.privacy_request import Consent as ConsentSchema
+from fidesops.ops.schemas.privacy_request import (
+    ConsentPreferences,
+    ConsentPreferencesWithVerificationCode,
+    ConsentRequestResponse,
+    VerificationCode,
+)
+from fidesops.ops.schemas.redis_cache import Identity
+from fidesops.ops.service._verification import send_verification_code_to_user
+from fidesops.ops.util.api_router import APIRouter
+from fidesops.ops.util.logger import Pii
+
+router = APIRouter(tags=["Consent"], prefix=V1_URL_PREFIX)
+
+logger = logging.getLogger(__name__)
+
+
+@router.post(
+    CONSENT_REQUEST,
+    status_code=HTTP_200_OK,
+    response_model=ConsentRequestResponse,
+)
+def create_consent_request(
+    *,
+    db: Session = Depends(get_db),
+    data: Identity,
+) -> ConsentRequestResponse:
+    """Creates a verification code for the user to verify access to manage consent preferences."""
+    if not config.redis.enabled:
+        raise FunctionalityNotConfigured(
+            "Application redis cache required, but it is currently disabled! Please update your application configuration to enable integration with a redis cache."
+        )
+
+    if not config.execution.subject_identity_verification_required:
+        raise FunctionalityNotConfigured(
+            "Subject identity verification is required, but it is currently disabled! Please update your application configuration to enable subject identity verification."
+        )
+
+    if not data.email:
+        raise HTTPException(HTTP_400_BAD_REQUEST, detail="An email address is required")
+
+    identity = ProvidedIdentity.filter(
+        db=db,
+        conditions=(
+            (ProvidedIdentity.field_name == ProvidedIdentityType.email)
+            & (ProvidedIdentity.hashed_value == ProvidedIdentity.hash_value(data.email))
+            & (
+                ProvidedIdentity.privacy_request  # pylint: disable=singleton-comparison
+                == None
```

Comment:
Instead of filtering on this relationship, which isn't an actual field, if you switch to the field, you can do something like `ProvidedIdentity.privacy_request_id.is_(None)` and get rid of this pylint ignore.